nilearn/image/__init__.py

```diff
@@ -15,5 +15,5 @@
            'smooth_img', 'crop_img', 'mean_img', 'reorder_img',
            'swap_img_hemispheres', 'concat_imgs', 'copy_img',
            'index_img', 'iter_img', 'new_img_like', 'threshold_img',
-           'math_img', 'load_img', 'clean_img',
+           'math_img', 'load_img', 'clean_img', 'pad_array',
```

Comment:
Do we want to make pad_array public? I am not sure. Functions in this module are to manipulate images, and not arrays.

Reply:
I'll make private, I think you're right